lib/config/validation.spec.ts

```diff
@@ -521,12 +521,11 @@ describe('config/validation', () => {
       expect(errors).toHaveLength(0);
     });
 
-    it('errors if aliases depth is more than 1', async () => {
+    it('errors if aliases have invalid url', async () => {
```

Comment:
any reason for test reorder?

Reply:
By mistake, got added at last instead if middle.